holmes/plugins/toolsets/azuremonitor_metrics/azuremonitor_metrics_toolset.py

```
@@ -0,0 +1,1306 @@
+                                        continue
+                    
+                    # Final debug log
+                    logging.debug(f"[FIRED_TIME_DEBUG] Final time_str for alert '{alert['alert_name']}': '{time_str}'")
+                    
+                    # Add monitor condition for more detailed status
+                    monitor_condition = alert.get('monitor_condition', 'Unknown')
+                    
+                    summary_lines.append(f"\n**{i}. {severity_icon} {alert['alert_name']}** {status_icon}")
+                    summary_lines.append(f"   🆔 **ALERT ID:** `{display_alert_id}`")
+                    summary_lines.append(f"   📋 **Full Alert Path:** `{actual_alert_id}`")
+                    summary_lines.append(f"   🔬 **Type:** Prometheus Metric Alert")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)